lagtraj/forcings/conversion/targets/dephy.py

```diff
@@ -571,6 +635,23 @@ def nudging_inv_time_prof(nudging_parameters, variable):
         "surfaceType": parameters.surfaceType,
         "surfaceForcing": parameters.surfaceForcing,
         "surfaceForcingWind": parameters.surfaceForcingWind,
+        "nudging_method_momentum": str(none_pass(parameters.nudging_method_momentum)),
+        "nudging_method_scalars": str(none_pass(parameters.nudging_method_scalars)),
+    }
+    ds_dephy.attrs.update(**dephy_dictionary)
+    nudging_specs_dict = {
+        "nudging_above_height_momentum": parameters.nudging_above_height_momentum,
```

Comment:
same as above I would turn this into a `for`-loop as each of these operations are the same